include/knowhere/comp/knowhere_check.h

```diff
@@ -38,6 +42,93 @@ SuppportMmapIndexTypeCheck(const std::string& index_name) {
         return false;
     }
 }
+
+inline bool
+CheckBooleanInJson(const knowhere::Json& json, std::string key, bool target) {
```

Comment:
any checks for `1` or `0` as well?
Something like `value == "1"` or `value == 1`

Reply:
This is consistent with the constraint represented by the bool in the configuration. Currently, something like 1 would be considered an invalid parameter.